ova/Ova2Ovf.py

```diff
@@ -30,7 +30,26 @@
 
 with tarfile.open(srcfile) as t:
     ovaFiles = t.getnames()
-    t.extractall()
+    def is_within_directory(directory, target):
+        
+        abs_directory = os.path.abspath(directory)
+        abs_target = os.path.abspath(target)
+    
+        prefix = os.path.commonprefix([abs_directory, abs_target])
+        
+        return prefix == abs_directory
+    
+    def safe_extract(tar, path=".", members=None, *, numeric_owner=False):
```

Comment:
Hello,

These changes use Python 3 syntax, this makes the building script fail in many systems that still use Python 2 as the default Python of the system, could you change the syntax so it also works with Python 2?